tests/unit/ceph/test_ceph_primary_handlers.py

```
@@ -0,0 +1,1061 @@
+    # Ensure lock decorators are passthroughs before importing
+    import tests.conftest  # noqa: F401
+
+    from tests.conftest import passthrough_lock
+    lock_mod = cast(object, importlib.import_module("zstacklib.utils.lock"))
+
+    setattr(lock_mod, "lock", passthrough_lock)
+    setattr(lock_mod, "file_lock", passthrough_lock)
+
+    module = importlib.import_module("cephprimarystorage.cephagent")
+    module = importlib.reload(module)
+except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)